edelphi-persistence/src/main/java/fi/metatavu/edelphi/dao/base/UserCreatedEntityDAO.java

```diff
@@ -0,0 +1,13 @@
+package fi.metatavu.edelphi.dao.base;
+
+import fi.metatavu.edelphi.domainmodel.users.User;
+
+import java.util.List;
+
+public interface UserCreatedEntityDAO<T> {
```

Comment:
Lets keep DAOs plain and simple, remove this abstraction and just implement dao lists case by case and remove this interface